src/test/ui/typeck/missing-private-fields-in-struct-literal.stderr

```diff
@@ -0,0 +1,23 @@
+error: cannot construct `S` with struct literal syntax due to private fields
+  --> $DIR/missing-private-fields-in-struct-literal.rs:13:13
+   |
+LL |     let _ = m::S {
+   |             ^^^^
+LL |         visible: true,
+LL |         a: (),
+   |         ----- private field
+LL |         b: (),
+   |         ----- private field
```

Comment:
Maybe just a note with something like "... and other private fields that were not provided" instead of pointing out the actual fields in the struct definition?